syntax = "proto3";
option cc_generic_services = true;
option java_generic_services = true;

import "common.proto";

package admin; 

message BackupChunk
{
    bool has_more = 1;
    bytes bytes = 2;
}

service AdminService 
{
   rpc StartBackup (common.EmptyMessage) returns (BackupChunk);
   rpc Next (common.EmptyMessage) returns (BackupChunk);
}

